docs/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 * State::handle_event can now handle multiple types of events. ([#887])
 * Added Named Component. ([#879])([#896])
 * Support for progressive jpeg loading. ([#877])
+* New `application_root_dir()` function in `amethyst_utils`. ([#831])
```

Comment:
Here you need to add a link to the pull request.
You need to add a line like this
```
 [#831]: https://github.com/amethyst/amethyst/pull/831
```